nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/impl/layers/convolution/Im2col.java

```diff
@@ -58,6 +58,13 @@ public Im2col(SameDiff sd, SDVariable input, Conv2DConfig config){
 
     public Im2col() {}
 
+    public Im2col(INDArray[] in, Conv2DConfig conv2DConfig) {
```

Comment:
Again, INDArray not INDArray[] (after fixing count in dl4j-dev-tools definition)